src/lib/components/editor/EditorToolbar.svelte

```diff
@@ -279,15 +287,11 @@
             </Tooltip>
         </div>
         <div class="flex">
-            <AiTranslateToolbarButton
-                {editor}
-                {canEdit}
-                {editableDisplayNameStore}
-                {resourceContent}
-                {machineTranslationStore}
-                {itemIndex}
-                bind:isLoading
-            />
+            {#if showMachineTranslationRating && !isLoading}
```

Comment:
Now that the logic for what is displayed is much simpler, we might be able to update the `widthRequired` var above to be more accurate. We had hardcoded 653 rather than getting granular with what the `AiTranslateToolbarButton` width was.